reformat-source.sh

```diff
@@ -7,8 +7,7 @@
 #
 # pip3 install yapf isort toml
 #
-if ! test -f odxtools/version.py || \
-   ! test -d tests || \
+if ! test -d tests || \
```

Comment:
I thought a bit about this and I came to the conclusion that instead of looking for the now autogenerated file `version.py`, we should use another essential one, e.g.,
```suggestion
if ! test -f odxtools/database.py || \
   ! test -d tests || \
```